types/aws-lambda/trigger/connect-contact-flow.d.ts

```diff
@@ -29,7 +29,7 @@ export interface ConnectContactFlowEvent {
     Name: "ContactFlowEvent";
 }
 
-export type ConnectContactFlowChannel = "VOICE" | "CHAT";
+export type ConnectContactFlowChannel = "VOICE" | "CHAT" | "EMAIL";
```

Comment:
Do the docs imply that `TASK` should also work? (The docs don't actually seeem to show any code...)